functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,94 @@
+# apply-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Apply setter values on resources fields. May set either the complete or partial field value.
+
+<!--mdtogo-->
+
+Setters provide a solution for template-free setting of field values. They are a
+safer alternative to other substitution techniques which do not have the context
+of the structured data. Setters may be invoked to modify the configuration
+using `apply-setters` function to set values.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt fn eval apply-setters:VERSION [DIR] -- [setter_name=setter_value]
+```
+
+Data model
+
+1. Fields reference setters specified as line comments -- e.g.
+
+```
+    # kpt-set: replicas
+```
+
+2. Input values to setters are provided as key-value pairs -- e.g.
+
+```
+    kpt fn eval apply-setters:unstable -- replicas=3
+```
+
+Control flow
```

Comment:
Why is this relevant to user of this function?

Reply:
Removed it.